.gitignore

```diff
@@ -12,3 +12,12 @@ project/plugins/project/
 .cache
 .lib/
 .bsp
+
+# Terraform
+**/.terraform/*
+*.tfstate
+*.tfstate.*
+*.tfvars
+*.tfvars.json
+crash.log
+crash.*.log
```

Comment:
*.plan?

Reply:
Is that a standard convention? I've not seen it before - I generated these with an IDE plugin that I think uses that official GitHub repo of technology-specific gitignores

Reply:
So I thought `terraform.plan` was the convention; apparently `tfplan` is the actual recommendation: https://www.terraform.io/cli/commands/plan#out-filename